deploy/example/example-app.yaml

```diff
@@ -51,6 +51,9 @@ spec:
       labels:
         app: my-csi-app
     spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 3000
```

Comment:
Would it be worth modifying the example further, to attempt to read the key, cert and ca files, maybe logging some information about them in a loop?

Reply:
Perhaps.. This should be in a follow up PR though

Reply:
I suggest that this is not added to the example as part of this PR. In my head, examples should be as simple as possible.